src/services/dashboards/dashboard-customize/modules/DashboardCustomizeSidebar.vue

```diff
@@ -178,17 +180,20 @@ $sidebar-md: 25%;
             @apply bg-blue-200;
         }
     }
-    .footer-wrapper {
-        @apply grid grid-cols-12 border-t border-gray-200;
-        position: fixed;
-        right: 0;
-        bottom: 0;
-        width: $(sidebar-md);
-        gap: 0.75rem;
-        padding: 0.75rem 1rem;
-        button {
-            @apply col-span-6;
-        }
+}
+.footer-wrapper {
+    @apply grid grid-cols-12 border-t border-gray-200;
+    width: 100%;
+    gap: 0.75rem;
+    padding: 0.75rem 1rem;
+    button {
+        @apply col-span-6;
     }
 }
 </style>
+<style lang="postcss">
+$footer-height: 57px;
+.p-sidebar .sidebar-wrapper {
+    padding-bottom: $footer-height;
+}
+</style>
```

Comment:
description: I use un-scoped style, because `p-sidebar` is out of module.

Reply:
It would be better with some annotations

Reply:
Okay! I'll add it!